### PR TITLE
Implements the inverse of decfa

### PR DESCRIPTION
**Pull request by arokem**
Just for you, @frheault.

### CODE REVIEW

File: dipy/io/utils.py [hunk 1]
```diff
@@ -91,3 +91,42 @@ def decfa(img_orig, scale=False):
     new_hdr.set_data_dtype(dest_dtype)
 
     return Nifti1Image(out_data, affine=img_orig.affine, header=new_hdr)
+
+
+def decfa_to_float(img_orig):
+    """
+    Create a nifti-compliant directional-encoded color FA file.
```
Comment (jchoude) [inline dipy/io/utils.py]: `file` -> `image`

File: dipy/io/utils.py [hunk 2]
```diff
@@ -91,3 +91,42 @@ def decfa(img_orig, scale=False):
...
+
+    Parameters
+    ----------
+    img_orig : Nifti1Image class instance.
+        Contains encoding of the DEC FA image with a 3D volume of data, where
+        each element is a (R, G, B) tuple in uint8.
+
+    Returns
+    -------
+    img : Nifti1Image class instance with uint8 dtype.
```
Comment (jchoude) [inline dipy/io/utils.py]: Written that it's going to be in `uint8`, but you convert to `np.float` in the header and the name of the function says so.

File: dipy/io/utils.py [hunk 3]
```diff
@@ -91,3 +91,42 @@ def decfa(img_orig, scale=False):
...
+
+
+    Notes
+    -----
+    For a description of this format, see:
+
+    https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1fields/nifti1fields_pages/datatype.html
```
Comment (jchoude) [inline dipy/io/utils.py]: Is it really there? This format is the classical format used by most software, but is not in the standard.

Comment (arokem) [reply dipy/io/utils.py]: I am not sure what you mean by "the standard". 

The triplet format is on this wiki page as "DT_RGB".

Comment (jchoude) [reply dipy/io/utils.py]: True. Went too fast.

**Comment by jchoude** [comment]
To me this is good to go. What's up with Travis acting up?

**Comment by arokem** [comment]
It ran into both https://github.com/nipy/dipy/issues/1752 and https://github.com/nipy/dipy/issues/1747 in one run. Doesn't look like anything to do with this PR.

**Comment by jchoude** [comment]
Perfect. Let's go then :rocket:

**Pull request merged**